usaspending_api/disaster/management/sql/disaster_covid19_file_d2_awards.sql

```diff
@@ -54,7 +54,8 @@ SELECT
     "latest_transaction"."recipient_location_state_name" AS "recipient_state_name",
     "latest_transaction"."recipient_location_zip5" AS "recipient_zip_code",
     "latest_transaction"."legal_entity_zip_last4" AS "recipient_zip_last_4_code",
-    "latest_transaction"."recipient_location_congressional_code" AS "recipient_congressional_district",
+    "latest_transaction"."recipient_location_congressional_code" AS "prime_award_summary_recipient_cd_original",
+    "latest_transaction"."recipient_location_congressional_code_current" AS "prime_award_summary_recipient_cd_original",
```

Comment:
I believe this should be `prime_award_summary_recipient_cd_current`

Reply:
Good catch